Cargo.toml

```diff
@@ -63,6 +63,7 @@ tokio-current-thread-01 = { package = "tokio-current-thread", version = "0.1", o
 
 [dev-dependencies]
 tokio-01 = { package = "tokio", version = "0.1" }
+futures-03 = { package = "futures", version = "0.3" }
```

Comment:
I don't think this dependency is needed, now that we've removed the calls to `futures_03::executor::block_on` from the tests.